store/tikv/txn.go

```diff
@@ -642,7 +642,7 @@ func (txn *KVTxn) LockKeys(ctx context.Context, lockCtx *tikv.LockCtx, keysInput
 		// PointGet and BatchPointGet will return value in pessimistic lock response, the value may not exist.
 		// For other lock modes, the locked key values always exist.
 		if lockCtx.ReturnValues {
-			val, _ := lockCtx.Values[string(key)]
+			val := lockCtx.Values[string(key)]
```

Comment:
In case that `ok == false`, val is the zero value for that type. It avoids panic. So this line may need to investigate if the owner is intended.

Reply:
OK, type is a struct, so i guess it is safe anyway.